staging/src/k8s.io/apimachinery/pkg/labels/selector_test.go

```diff
@@ -617,3 +617,16 @@ func TestSafeSort(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkSelectorFromValidatedSet(b *testing.B) {
+	set := map[string]string{
+		"foo": "foo",
+		"bar": "bar",
+	}
+
```

Comment:
NIT: b.ResetTimer()

Reply:
Not needed - we only create one small map before - so that is negligible.